docs/en/security-privacy-considerations.rst

```diff
@@ -0,0 +1,499 @@
+the attacker might have access to the presentation contained in the VP Token. Given that, the proper implementation of TLS guarrantes confidentiality, avoiding the leakage of the response. 
+In addition to TLS, the existing implementation of **nonce** and audience checks in the presentation protocol should help to meet the P-41 security requirement. 
+Regarding the **nonce** claim, the Verifier MUST check that the **nonce** value in the VP Token matches the **nonce** value that is created by the Verifier during the authorization request.
+Regarding the **aud** value, the Verifier must check that the audience of the presentation matches the Verifier's identifier. 
+
+SR-P-50
+~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |partially-check-icon| 
+     - The protocol must ensure that third parties cannot interfere with the binding process.
+
+In the issuance phase, the Holder binding happens at the Credential request to the protected Credential endpoint. This means that the attacker needs to obtain the access token 
+first and therefore send the request to the Credential endpoint and bind the Credentials to the keys under his control. The IT Wallet specification requires the use of a sender-constrained 
+access token, which means that the access token binds to the device using cryptographic materials. 
+
+The second surface for the attack is related to key management. In the case of using software-based keys, it is possible to clone the keys and move them to a device under 
+attacker control, and in the case of stealing the Credentials as well, the attacker can easily create proof of possession of the keys. IT Wallet is less vulnerable to these attacks as it supports local 
+internal WSCD that uses hardware-based keys. However, the lack of a 
+certification profile that certifies the local internal WSCD against highly capable attackers (the certification for current TEE solutions on the market reaches AVA_VAN.3 at most 
+as shown for example in https://www.tuv-nederland.nl/assets/files/cerfiticaten/2021/08/nscib-cc-0244671-cr-1.0.pdf or https://globalplatform.org/specs-library/tee-protection-profile-v1-3/) makes the requirement only partially satisfied.
+
+.. note::
+   In the EUDI Wallet context, the local internal WSCD and other WSCD deployments are still under certification 
+   (see https://docbox.etsi.org/ESI/Open/workshops/202409_CEN_ETSI_Workshop/DAY3-8%20Certification%20for%20EU%20Digital%20Identity%20Wallets/DAY3-8-26%20ETSI_CEN_WS_WSCA%20Jan%20Kjaersgaard.pdf).
+
+SR-V-10
+~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |partially-check-icon|
+     - (conditional w.r.t  I-50+V-20) The Verifier must ensure that the Credential was stored in a secure Wallet.
+
+Verifier checks the Wallet Attestation during exchanges (sent with the authorization response), ensuring that it meets the security criteria required by the Verifier and that it is issued by a trusted Wallet Provider.  
```

Comment:
```suggestion
Verifier checks the Wallet Attestation during exchanges (sent with the authorization response), ensuring that it meets the security criteria required by the Verifier and under the sole responsability of its issuer, the trusted Wallet Provider.  
```

Reply:
The verb is missing. Here the update text:
> Verifier checks the Wallet Attestation during exchanges (sent with the authorization response), ensuring that it meets the security criteria required by the Verifier and is under the sole responsability of its issuer, the trusted Wallet Provider.